src/api/RequestApi.ts

```diff
@@ -257,6 +255,7 @@ export type IInRequest = {
   supGovLead: SPPersona;
   /** Required - The employee GAL entry. If the user doesn't exist yet, then it will be undefined */
   employee: SPPersona | undefined;
+  isTraveler: "yes" | "no";
```

Comment:
You didn't move the comment with this:
/** Required - Can only be 'true' if it is a Civ/Mil.  Must be 'false' if it is not a 'civ' or 'mil' */